src/test/java/org/folio/oaipmh/service/impl/InstancesServiceImplTest.java

```diff
@@ -0,0 +1,235 @@
+import javax.ws.rs.NotFoundException;
+import java.time.OffsetDateTime;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+import static org.folio.rest.impl.OkapiMockServer.OAI_TEST_TENANT;
+import static org.folio.rest.jooq.Tables.INSTANCES;
+import static org.folio.rest.jooq.Tables.REQUEST_METADATA_LB;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@ExtendWith(VertxExtension.class)
+class InstancesServiceImplTest extends AbstractInstancesTest {
+
+  private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+  private static final String TEST_TENANT_ID = "oaiTest";
+  private static final int mockPort = NetworkUtils.nextFreePort();
+
+  private static final int EXPIRED_REQUEST_IDS_EMPTY_LIST_TIME = 2000;
+  private static final int ZERO_EXPIRED_INSTANCES_TIME = INSTANCES_EXPIRATION_TIME_IN_SECONDS * 2;
+
+  private static final String REQUEST_ID_DAO_ERROR = "c75afb20-1812-45ab-badf-16d569502a99";
+  private static final String REQUEST_ID_DAO_DB_SUCCESS_RESPONSE = "c86afb20-1812-45ab-badf-16d569502a99";
+
+  private static List<String> validRequestIds = Collections.singletonList(REQUEST_ID_DAO_DB_SUCCESS_RESPONSE);
+  private static List<String> daoErrorRequestId = Collections.singletonList(REQUEST_ID_DAO_ERROR);
+
+  @Autowired
+  private InstancesDao instancesDao;
+  @Autowired
+  private InstancesServiceImpl instancesService;
+
+  @BeforeAll
+  void setUpClass(Vertx vertx, VertxTestContext testContext) throws Exception {
+    logger.info("Test setup starting for " + TestUtil.getModuleId());
+    PostgresClient.getInstance(vertx)
+      .startEmbeddedPostgres();
+    TestUtil.prepareDatabase(vertx, testContext, OAI_TEST_TENANT, List.of(INSTANCES, REQUEST_METADATA_LB));
+    Context context = vertx.getOrCreateContext();
+    SpringContextUtil.init(vertx, context, ApplicationConfig.class);
+    SpringContextUtil.autowireDependencies(this, context);
+    new OkapiMockServer(vertx, mockPort).start(testContext);
+    LiquibaseUtil.initializeSchemaForTenant(vertx, TEST_TENANT_ID);
+    testContext.completeNow();
+  }
+
+  @AfterAll
+  static void tearDownClass(Vertx vertx, VertxTestContext testContext) {
+    PostgresClientFactory.closeAll();
+    vertx.close(testContext.succeeding(res -> {
+      PostgresClient.stopEmbeddedPostgres();
+      testContext.completeNow();
+    }));
+  }
+
+  @Test
+  void shouldReturnFutureWithEmptyList_whenThereNoExpiredRequestIds(VertxTestContext testContext) {
+    testContext.verify(() -> instancesService.cleanExpiredInstances(TEST_TENANT_ID, ZERO_EXPIRED_INSTANCES_TIME)
+      .onComplete(testContext.succeeding(ids -> {
+        assertTrue(ids.isEmpty());
+        testContext.completeNow();
+      })));
+  }
+
+  @Test
+  void shouldReturnFutureWithExpiredIds_whenThereExpiredRequestIdsArePresented(VertxTestContext testContext) {
```

Comment:
I think the word "there" is not necessary for this method name

Reply:
I am agree